plugins/source/aws/resources/services/apigateway/vpc_links_fetch.go

```diff
@@ -26,6 +29,11 @@ func fetchApigatewayVpcLinks(ctx context.Context, meta schema.ClientMeta, parent
 func resolveApigatewayVpcLinkArn(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
 	cl := meta.(*client.Client)
 	link := resource.Item.(types.VpcLink)
-	arn := cl.RegionGlobalARN(client.ApigatewayService, "/vpclinks", *link.Id)
-	return resource.Set(c.Name, arn)
+	return resource.Set(c.Name, arn.ARN{
+		Partition: cl.Partition,
+		Service:   string(client.ApigatewayService),
+		Region:    cl.Region,
+		AccountID: "",
+		Resource:  fmt.Sprintf("/vpclinks/%s", aws.ToString(link.Id)),
```

Comment:
This looks the same as before, but it's a little suspicious that it's the only `Resource` that starts with a forward slash :suspect: